src/core/server/base.js

```diff
@@ -269,9 +269,8 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
 
 export function runServer({ listen = true, app = appName } = {}) {
   if (!app) {
-    log.fatal(
+    throw new Error(
       `Please specify a valid appName from ${config.get('validAppNames')}`);
-    process.exit(1);
```

Comment:
I didn't want the test suite to be completely killed on this line: https://github.com/mozilla/addons-frontend/blob/master/tests/server/amo/TestDetailsPage.js#L16 I'd like it to fail with the error instead. Actually, I can't really tell if `process.exit(1)` was the right thing to do. When I add that back, `npm run dev:amo` just keeps spawning the process in a loop as if it's trying to start it again. Hmm.